src/components/screens/vacation-requests-screen.tsx

```diff
@@ -78,6 +100,10 @@ const VacationRequestsScreen = () => {
     }
   };
 
+  const toggleIsUpcoming = () => {
```

Comment:
Seems basic but all function should have TSDocs so in this case it would be something like
/**
 * Handler for upcoming/ past vacations toggle click
 */